sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -13,12 +13,17 @@ namespace Azure.Communication
     /// </summary>
     public class CommunicationTokenRefreshOptions
     {
+        /// <summary>Default proactive refreshing interval in minutes.</summary>
+        public const int DefaultExpiringOffsetMinutes = 5;
```

Comment:
I don't understand why some of the fields here are public, some internal, some private and there seems to be no order in it. 
Could you please:
1) make sure the accessibility level is set correctly for each of the members
2) group the members by their accessibility level 
3) add comments even to less accessible members. it will do no harm to have them there + it'll be consistent.

Reply:
Have we considered how we would make this constant public in other languages? In TypeScript for example, `CommunicationRefreshOptions` are an interface, so we would have to export a new constant.

If we want Contoso to know the default, we can consider mentioning the default time in a comment instead.

Reply:
If this is only made public now so that it can be used in the tests, let's make it `internal` instead.

Reply:
This is a functional change to go from 10min to 5min. Are we only changing that to satisfy MSAL?
Will we change the default time across all languages?

Reply:
I'd revert it back to 10, make it internal, add the default to the comment + to the [docs](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/communication.common-readme?view=azure-dotnet).